source/partials/drupal-8-convert-to-composer.md

```diff
@@ -0,0 +1,255 @@
+
+This script is provided without warranty or direct support. Issues and questions may be filed in GitHub but their resolution is not guaranteed.
+
+Proceed at your own risk. Automation is better when you understand what it's doing.
+
+- The script does not resolve `composer.json` version problems.
+
+  - If there's a version conflict when you run `composer install`, you will need to resolve the conflicts yourself.
+
+- The script does not check for Drupal 9 compatibility.
+
+  - Use the [Upgrade Status](https://drupal.org/project/upgrade_status) module to check for Drupal 9 compatibility.
+
+- The script does not resolve module stability.
+
+  - If you get the following error, the module listed in the error (or its dependencies) does not meet compatibility requirements:
+
+   ```none
+   [InvalidArgumentException]
+   Could not find a version of package drupal/MODULE_NAME matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
+   ```
+
+   If there is no stable version you can switch to, you may need to adjust the `minimum-stability` setting of `composer.json` to a more relaxed value, such as `beta`, `alpha`, or even `dev`. You can read more about `minimum-stability` in the [Composer documentation](https://getcomposer.org/doc/04-schema.md#minimum-stability).
+
+- The script does not resolve patches.
+
+  Many times a module will be "patched" or have a `.patch` file that fixes known issues before the fix is available in the downloaded version. This script does not attempt to resolve any patches.
+
+If you still want to try it:
+
+1. Say the following out loud: "Pantheon is not responsible for what I am about to do."
+
+1. Create a directory called `bin` within your repository and copy the `migrateModules.php` file from [GitHub](https://github.com/pantheon-upstreams/drupal-project/pull/8/files#diff-c68470275758ca395b98d53ed258b63519435d492dd51531c4cf372814c6593e) into that directory.
+
+1. To run the module migration script, `cd` to the root directory of the repository and run `bin/migrateModules.php`.
+
+1. To see which modules were added, run `git diff composer.json`.
+
+1. Remove the `composer.lock` file. Composer will create a new one with the modules you just added.
+
+1. Run `composer install` and resolve any remaining version conflicts.
+
+</Accordion>
+-->
+
+#### Libraries
+
+Libraries can be handled similarly to modules, but the specifics depend on how your library code was included in the source site. If you're using a library's API, you may have to do additional work to ensure that library functions properly.
+
+### Custom Code
+
+Manually copy custom code from the existing site repository to the Composer-managed directory.
+
+#### Modules and Themes
+
+Modules:
+
+```bash{promptUser:user}
+mkdir -p ../$SITE-composer/web/modules/custom # create the directory if it doesn't already exist
+cp -r modules/custom/awesome_module ../$SITE-composer/web/modules/custom
```

Comment:
Instead of the lines above, use:
```
git checkout master modules/custom
git mv modules/custom web/modules/
git commit -m "Copy custom modules"
```

Reply:
commands replaced